src/server/utilities/cspHeader/index.js

```diff
@@ -270,8 +297,26 @@ const directives = {
       'https://passport-control.int.tools.bbc.co.uk/bookmarkletScript.js', // Passport bookmarklet - int
       'https://passport-control.test.tools.bbc.co.uk/bookmarkletScript.js', // Passport bookmarklet - test
       'https://passport-control.tools.bbc.co.uk/bookmarkletScript.js', // Passport bookmarklet - live
+      // dotcom required urls
+      'https://tpc.googlesyndication.com',
+      'https://privacy.crwdcntrl.net',
+      'https://gn-web-assets.api.bbc.com',
+      'https://securepubads.g.doubleclick.net',
+      'https://bcp.crwdcntrl.net',
+      'https://ad.crwdcntrl.net',
+      'https://sb.scorecardresearch.com',
+      'https://me-ssl.effectivemeasure.net',
+      'https://bbc.gscontxt.net',
+      'https://adservice.google.co.uk',
+      'https://adservice.google.com',
+      'https://t.effectivemeasure.net',
+      'https://tags.crwdcntrl.net',
+      'https://collector.effectivemeasure.net',
+      'https://cdn.ampproject.org',
+      // end dotcom required urls
       "'self'",
       "'unsafe-inline'",
+      "'unsafe-eval'",
```

Comment:
I'd be very wary of putting `unsafe-eval` in the live CSP, and should be one of the issues that we raise with GNL.